x-pack/plugins/cases/common/api/cases/case.test.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { ConnectorTypes } from '../connectors';
+import { ConnectorTypes } from '../../types/domain/connector/v1';
```

Comment:
How about importing `{ connectorDomainV1 } from '../../types/domain/connector;`?

`index` exports it anyway so I guess we might as well use it.

Reply:
This will change in another PR very soon because the case will also be versioned. Do you mind if we leave it as it is for the moment?